common.sh

```diff
@@ -339,7 +339,7 @@ readonly COREOS_EPOCH=1372636800
 TODAYS_VERSION=$(( (`date +%s` - ${COREOS_EPOCH}) / 86400 ))
 
 # Download URL prefix for SDK and board binary packages
-if [[ "${FLATCAR_BUILD_ID}" =~ ^nightly-.*$ ]] ; then
+if [[ -n "${FLATCAR_BUILD_ID}" ]] ; then
```

Comment:
This also means setting up the binary package URL for the boards, or? It's a tricky question but I don't think we want to use binary packages by default for all dev builds because doing a fix and iterating over it with a rebuild would not result in the fix being picked up because the binary package takes precedence over building from source. The developer would have to bump the ebuild version each time, that's such a big trap.

Reply:
(The URL gets used in `get_board_binhost`/`get_sdk_binhost` from `build_library/toolchain_util.sh` to set up the binpkg URLs)

Reply:
I have taken up a different approach, currently running CI in http://jenkins.infra.kinvolk.io:8080/job/container/job/sdk/512/cldsv. Let's see.